pkg/controller/node/status.go

```diff
@@ -136,18 +137,44 @@ func isNodeManaged(node *corev1.Node) bool {
 	if node.Annotations == nil {
 		return false
 	}
-	cconfig, ok := node.Annotations[daemonconsts.CurrentMachineConfigAnnotationKey]
-	if !ok || cconfig == "" {
-		return false
+
+	cconfig, hasConfig := node.Annotations[daemonconsts.CurrentMachineConfigAnnotationKey]
+	cimage, hasImage := node.Annotations[daemonconsts.CurrentImageConfigAnnotationKey]
+
+	// if we have an image config present
+	if hasImage && cimage != "" {
+		return true
 	}
```

Comment:
Things like this are why I was previously  hesitant to have a completely separate set of annotations -- I didn't want our changes to be too invasive -- there are a bunch of places where we assume that the presence/absence of `daemonconsts.CurrentMachineConfigAnnotationKey` means something.

Reply:
Yeah...fair enough.  Hmm. 
So definitely a bit more risk to a master merge with this.  I think we can do it though, just needs some auditing. I'm also pretty confident our e2es cover this.